utils/errors.go

```diff
@@ -26,3 +26,13 @@ type KeyLengthError struct{}
 func (e KeyLengthError) Error() string {
 	return "invalid uuid length"
 }
+
+/**************************/
+/* Put errors			  */
+/**************************/
```

Comment:
Nitpick: I think this file is small enough that we don't need these block style comments.

Reply:
Removed